go/staking/tests/tester.go

```diff
@@ -583,7 +583,7 @@ func testEscrowEx( // nolint: gocyclo
 	newDstAcc = nil
 
```

Comment:
There should also be some tests for all the new query methods.

Reply:
Done in #3774.